test/expression.txt

```diff
@@ -517,3 +517,130 @@ PromQL(
     )
   )
 )
+
+# Simple At start
+
+foo @ start()
```

Comment:
Could you add tests (and if necessary, code) for whitespace around the parens, like `foo @ start ( )`? Those should work in Prometheus itself, as the parentheses are separate tokens. Maybe they should be considered separate tokens here as well?

Reply:
Maybe it will be easier to consider them as a separate token indeed.